docs/development/build-instructions-macos.md

```diff
@@ -13,6 +13,45 @@ Follow the guidelines below for building **Electron itself** on macOS, for the p
 * [node.js](https://nodejs.org) (external)
 * Python >= 3.7
 
+### Arm64-specific prerequisites
+
+* Rosetta 2
+  * this can be installed by using the softwareupdate command line tool.
```

Comment:
I'm not sure if Rosetta is strictly required to run on ARM, could we confirm that? I might say something like:
```suggestion
  * We recommend installing Rosetta if using dependencies that need to cross-compile on x64 and arm64 machines. Rosetta can be installed by using the `softwareupdate` command line tool:
```